readme.md

```diff
@@ -71,22 +64,111 @@ open_feature_api.set_provider(NoOpProvider())
 open_feature_client = open_feature_api.get_client()
 ```
 
-## Contacting us
+### Basics:
 
-We hold regular meetings which you can see [here](https://github.com/open-feature/community/#meetings-and-events).
+While Boolean provides the simplest introduction, we offer a variety of flag types.
 
-We are also present on the `#openfeature` channel in the [CNCF slack](https://slack.cncf.io/).
+```python
+# Depending on the flag type, use one of the methods below
+flag_key = "PROVIDER_FLAG"
+boolean_result = open_feature_client.get_boolean_value(key=flag_key,default_value=False)
+integer_result = open_feature_client.get_integer_value(key=flag_key,default_value=-1)
+float_result = open_feature_client.get_float_value(key=flag_key,default_value=-1)
+string_result = open_feature_client.get_string_value(key=flag_key,default_value="")
+object_result = open_feature_client.get_object_value(key=flag_key,default_value={})
+```
+
+You can also bind a provider to a specific client by name instead of setting that provider globally:
+
+```python
+
+open_feature_api.set_provider(NoOpProvider())
+```
+
+Each provider class may have further setup required i.e. secret keys, environment variables etc
+
+<!-- TODO: example of named client binding -->
+### Context-aware evaluation:
+
+Sometimes the value of a flag must take into account some dynamic criteria about the application or user, such as the user location, IP, email address, or the location of the server.
+In OpenFeature, we refer to this as [`targeting`](https://openfeature.dev/specification/glossary#targeting).
+If the flag system you're using supports targeting, you can provide the input data using the `EvaluationContext`.
+
+```python
+from open_feature.open_feature_api import get_client, get_provider, set_provider
+from open_feature.open_feature_evaluation_context import (
+    api_evaluation_context,
+    set_api_evaluation_context,
+)
+
+global_context = EvaluationContext(
+    targeting_key="targeting_key1", attributes={"application": "value1"}
+)
+request_context = EvaluationContext(
+    targeting_key="targeting_key2", attributes={"email": request.form['email']}
+)
+
+## set global context
+set_api_evaluation_context(first_context)
+
+# merge second context
+client = get_client(name="No-op Provider", version="0.5.2")
+client.get_string_value("email", None, request_context)
+
+```
+### Events
+
+Events allow you to react to state changes in the provider or underlying flag management system, such as flag definition changes, provider readiness, or error conditions.
+
```

Comment:
I would recommend either adding a section saying that `Events` haven't been implemented and possibly adding a link to a GitHub issue or removing this section for now.

Reply:
I added an events issue and linked it, as well as linking the hooks GitHub issue as those are currently unimplemented as well.